app/src/test/java/io/apicurio/registry/storage/MultitenantRegistryProfile.java

```diff
@@ -13,29 +13,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+package io.apicurio.registry.storage;
 
-package io.apicurio.registry.mt;
+import java.util.Collections;
+import java.util.Map;
 
-import javax.enterprise.context.ApplicationScoped;
-import javax.enterprise.inject.Produces;
-import javax.inject.Inject;
-
-import io.apicurio.registry.types.Current;
+import io.quarkus.test.junit.QuarkusTestProfile;
 
 /**
- * @author eric.wittmann@gmail.com
+ * @author Fabian Martinez
```

Comment:
GitHub thinks you stole this from me. lol

Reply:
lol